src/store/modules/media.ts

```diff
@@ -77,7 +69,7 @@ export default {
       }
 
       const compoundsPromise = axios
-        .get<Compound[]>(`${settings.apis.warehouse}/bioentities/compounds`)
+        .get<Compound[]>(`${settings.apis.warehouse}/media/compounds`)
```

Comment:
The `Compound` type should probably be called `MediumCompound` - the `mass_concentration` field is specific to media